src/ducks/apps/index.js

```diff
@@ -238,10 +238,11 @@ export function uninstallApp (slug) {
   }
 }
 
-export function installApp (slug, source) {
+export function installApp (slug, source, isUpdate = false) {
```

Comment:
Should n't be more logical to have a method called `updateApp`, instead of passing a boolean parameter to a method name `installApp` ?

Reply:
Since the update is just a change of `HTTP verb` I am really not sure about using another function for that. I would prefer to rename this method to `installOrUpdateApp`. What do you think?

Reply:
We could use 3 functions, but the main one should be name `installApp`, so let's go with the actual solution 👍